app/controllers/admin/api/application_plan_limits_controller.rb

```diff
@@ -14,15 +14,19 @@ class Admin::Api::ApplicationPlanLimitsController < Admin::Api::BaseController
   ##~ op.description = "Returns the list of all limits associated to an application plan."
   ##~ op.group = "application_plan_limits"
   #
+  ##~ @parameter_per_page_50 = {:name => "per_page", :description => "Number of results per page. Default and max is 50.", :dataType => "int", :paramType => "query", :defaultValue => "50"}
```

Comment:
Note to us: https://github.com/3scale/porta/blob/master/app/controllers/admin/api/accounts_controller.rb#L26 is 500